ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -17,6 +18,12 @@
 # --------------- #
 
 
+def _avoid_torch_unsupported_cases(x1, x2, dtype):
+    if "64" not in dtype[0]:
+        if x1.ndim == 1 or x2.ndim == 1:
+            dtype[0] = "float64" if "float" in dtype[0] else "int64"
+
+
```

Comment:
I'm not sure if I follow this. Can you explain why this is needed?

Reply:
As discussed in this [message](https://discord.com/channels/799879767196958751/1028303853315162112/1153082103710363839)

```python
x1=torch.tensor([0], dtype=torch.uint8) 
x2 = torch.tensor([1], dtype=torch.uint8)
torch.tensordot(x1, x2, dims=1)
```
```
RuntimeError: expected scalar type Long but found Byte
```
pytorch does not support tensordot with scaler values for with dtypes less than 64 bit so I modified the test to avoid them

Reply:
Got it, thanks for explaining again!